source/production/F0.Minesweeper.Components/Layout/Footer.razor

```diff
@@ -1,7 +1,38 @@
 <footer id="f0-footer">
-    <hr />
-    <span>Follow this project on </span>
-    <a href="https://github.com/Flash0ver/F0.Minesweeper" target="_blank">
-        GitHub
-    </a>
+	<hr />
+	<span>Follow this project on </span>
+	<a href="https://github.com/Flash0ver/F0.Minesweeper" target="_blank">
+		GitHub
+	</a>
+
+	@if (ProductName is not null && ProductVersion is not null)
+	{
+		<p>@ProductName v@(ProductVersion)</p>
+	}
+	@if (FrameworkVersion is not null)
+	{
+		<p>powered by @FrameworkVersion</p>
+	}
+	@if (CopyrightNotice is not null)
+	{
+		<p>
+			@CopyrightNotice
+			<a href="https://github.com/Flash0ver/F0.Minesweeper/graphs/contributors" target="_blank">Contributors</a>
+			(<a href="https://github.com/Flash0ver/F0.Minesweeper/blob/main/LICENSE" target="_blank">License</a>)
+		</p>
+	}
 </footer>
+
+@code {
+	[Parameter]
```

Comment:
Since we derive the `Productname`, `ProductVersion` and `FrameworkVersion` from the executing assembly it kind of feels wrong that these informations are a parameter since they can not be changed.

An option would be to read the _executing assembly_ at startup, register it as e.g. a `IVersionInfoService` in the DI container and inject it here via `[Inject]`